libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -150,30 +150,12 @@ const AP_Param::GroupInfo AP_Proximity::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_IGN_WID6", 15, AP_Proximity, _ignore_width_deg[5], 0),
 
-#if PROXIMITY_MAX_INSTANCES > 1
-    // @Param: 2_TYPE
-    // @DisplayName: Second Proximity type
-    // @Description: What type of proximity sensor is connected
-    // @Values: 0:None,7:LightwareSF40c,1:LightWareSF40C-legacy,2:MAVLink,3:TeraRangerTower,4:RangeFinder,5:RPLidarA2,6:TeraRangerTowerEvo,8:LightwareSF45B,10:SITL,12:AirSimSITL
-    // @User: Advanced
-    // @RebootRequired: True
-    AP_GROUPINFO("2_TYPE", 16, AP_Proximity, _type[1], 0),
-
-    // @Param: 2_ORIENT
-    // @DisplayName: Second Proximity sensor orientation
-    // @Description: Second Proximity sensor orientation
-    // @Values: 0:Default,1:Upside Down
-    // @User: Standard
-    AP_GROUPINFO("2_ORIENT", 17, AP_Proximity, _orientation[1], 0),
-
-    // @Param: 2_YAW_CORR
-    // @DisplayName: Second Proximity sensor yaw correction
-    // @Description: Second Proximity sensor yaw correction
-    // @Units: deg
-    // @Range: -180 180
+    // @Param{Copter}: _LAND_DETECT
```

Comment:
How about we renamed this parameter to "_IGN_GND"?  Let's rename "_land" in the methods to "_ground" as well.